src/test/integration/api_request.py

```diff
@@ -0,0 +1,53 @@
+import json
+import requests
+import urllib2
+import ssl
+from configuration_load import *
+
+
+def good_request():
+    query_params = {'q': 'Oxford'}
+    headers = {'Authorization': get_access_token()}
+    request = requests.get(get_url(), params=query_params, headers=headers)
+    return request.status_code
```

Comment:
The links attribute should be present all the time. The only difference with a query with more than 10 results is that "next" will not be null. I am adding tests for links assertions.